sharding-proxy/sharding-proxy-transport/sharding-proxy-transport-postgresql/src/main/java/org/apache/shardingsphere/shardingproxy/transport/postgresql/packet/command/query/binary/bind/protocol/PostgreSQLStringBinaryProtocolValue.java

```diff
@@ -33,7 +33,10 @@ public int getColumnLength(final Object value) {
     
     @Override
     public Object read(final PostgreSQLPacketPayload payload) {
-        return payload.readStringNul();
+        payload.getByteBuf().readerIndex(payload.getByteBuf().readerIndex() - 4);
```

Comment:
Remove payload.readInt4() in getParameters(), then this line can be deleted.

Reply:
Will it affect other `PostgreSQLBinaryProtocolValue` if remove payload.readInt4() in getParameters()?

Reply:
Affect which PostgreSQLBinaryProtocolValue?